src/components/button1.tsx

```diff
@@ -1,5 +1,40 @@
-export default function Button1() {
+"use client"
+import { useState } from "react";
+
+export default function Button1(label: string) {
+        
+    
+    label = "Sidrah and Oliver";
+
+    const [tempTotal, setTempTotal] = useState("");
+    const [shortForecast, setShortForecast] = useState();
+    
+    const handleClick = () => {
+        fetch("https://api.weather.gov/gridpoints/BOX/69,92/forecast")
+            .then((res) => res.json())
+            .then((json) => {
+                setTempTotal("Current temperature: " + json.properties.periods[0].temperature + json.properties.periods[0].temperatureUnit);
+                setShortForecast(json.properties.periods[0].shortForecast);
+            });
```

Comment:
Great work! One thing to think about...what happens if these requests fail? How might we handle errors without presenting an ugly error message to the user?